src/components/validations/form-with-validations.hoc.js

```diff
@@ -13,14 +13,13 @@ const withValidations = (WrappedComponent) => {
       infoCount: 0
     }
 
-    static propTypes = {
-      children: PropTypes.node // Children elements
-    }
-
     inputs = {};
 
-    addInput = (name, validate) => {
-      this.inputs[name] = validate;
+    addInput = (name, validate, value) => {
+      this.inputs[name] = {
+        validation: validate,
+        value
```

Comment:
This doesn't look right?

Reply:
No you're right, leftover from another solution I was trying to implement, removed in